viewtopic.php

```diff
@@ -456,7 +456,7 @@
 	echo "\t\t\t\t\t\t".'<label>';
 
 ?>
-<textarea name="req_message" rows="7" cols="75" tabindex="<?php echo $cur_index++ ?>"></textarea></label>
+<textarea dir="auto" name="req_message" rows="7" cols="75" tabindex="<?php echo $cur_index++ ?>"></textarea></label>
```

Comment:
Out of interest: why are changes like these only made to `textarea` tags, but no `input` elements?

Reply:
If we want to solve this problem everywhere, then it should be applied many places. My changes addressed the most serious issues.